js/src/api/contract/contract.spec.js

```diff
@@ -41,12 +41,25 @@ describe('api/contract/Contract', () => {
       type: 'function', name: 'test2',
       outputs: [{ type: 'uint' }, { type: 'uint' }]
     },
-    { type: 'constructor' },
+    {
+      type: 'constructor',
+      inputs: [{ name: 'boolin', type: 'bool' }, { name: 'stringin', type: 'string' }]
```

Comment:
Cool. Small grumble... 

Would have preferred to keep a non-parameters constructor in addition to the new with-parameters constructor (i.e. separate tests). The no-parameters tests obviously worked (maybe named incorrectly), it was the with-parameters case that was missing. 

Basically if things ever go haywire with some extra stuff added to constructors where we expect none, we won't catch it.